src/ResolvesResources.php

```diff
@@ -92,15 +92,14 @@ protected function collectDeferredValues(array &$resource): array
      */
     protected function resolveDeferredValues(array $deferredValues): void
     {
-        collect($deferredValues)->groupBy(function (DeferredValue $deferredValue) {
+        Collection::make($deferredValues)->groupBy(function (DeferredValue $deferredValue) {
             return $deferredValue::class;
-        })->each(function (Collection $deferredValues, $deferredValueClass) {
+        })->each(function (Collection $deferredValues, /** @param class-string<DeferredValue> */ string $deferredValueClass) {
```

Comment:
Ooh! I never thought of in-lining that! 👍

Reply:
Glad I could share something new!